app/message/delete-metadata-from-cosmos.js

```diff
@@ -0,0 +1,29 @@
+const Wreck = require('@hapi/wreck')
+const { serverConfig } = require('../config')
+
+const deleteMetadataFromCosmos = async (request, blobReference) => {
+  console.log('Deleting metadata from Cosmos:', blobReference)
+  try {
+    const mutation = `mutation DeleteFileMetadataByBlobReference {
+    deleteFileMetadataByBlobReference(blobReference: "${blobReference}") {
+        code
```

Comment:
If we're not using any of these returned properties (lines 9-11) from the mutation then remove them. 

Consider if we should be using any of them. eg to do a check on the returned status code or return the message if there is an error.